src/components/datatypes/CodeableConcept.js

```diff
@@ -15,7 +15,7 @@ const CodeableConcept = props => {
 };
 
 CodeableConcept.propTypes = {
-  fhirData: PropTypes.shape({}).isRequired,
+  fhirData: PropTypes.array.isRequired,
```

Comment:
How about a stricter check? :
```suggestion
  fhirData: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
```